monitoring.adoc

```diff
@@ -790,17 +790,27 @@ This pattern consists of 6 blocks of data:
 * Thread number with a prefix (e.g. `[qtp1043351526-547:https:https://localhost:7071/service/admin/soap/DeleteAccountRequest]`, `[Index-9]`, etc)
 * {product-name} context
 * Component name (e.g. `soap`, `mailbox`, `mbxmgr`, etc)
-* Log message
+* Log message. *Note:* the log message section may span multiple lines. When a log message contains an exception, the stack trace will always start on a new line below the error message.
 
 You can read more about Log4j patterns in https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/PatternLayout.html[Log4j PatternLayout documentation].
 
 *Thread prefix in mailbox.log*
 
-Thread numbers in mailbox.log are prefixed to identify internal components. The following prefixes are currently used in {product-name}: `btpool`, `pool`, `LmtpServer`, `ImapServer`,`ImapSSLServer`, `Pop3Server`, `Pop3SSLServer`, `ScheduledTask`, `Timer`, `AnonymousIoService`, `CloudRoutingReaderThread`, `GC`, `SocketAcceptor`, `Thread`, `qtp`.
+Thread numbers in mailbox.log are prefixed to identify internal components. Most threads have the following naming convention: "*{thread prefix}-{thread number}*".
```

Comment:
Maybe most threads have one of the following naming conventsion: {thread-prefix}-{thread number} or  {thread prefix}-{thread number}:{url}. In my log parser I have this currently:

    if ($line->{thread} =~ /^(.*)-(\d+):(.*)$/) {
      $line->{thread} = { name=>$1,number=>$2,url=>$3 };
    } elsif ($line->{thread} =~ /^(.*)-(\d+)$/) {
      $line->{thread} = { name=>$1,number=>$2 };
    } else {
      $line->{thread} = { name=>$line->{thread} };
    }

With the above this is what gets reported in solr for the thread.name
      "thread.name":[
        "ImapServer",5548385,
        "Pop3Server",2053677,
        "LmtpServer",350402,
        "qtp1813406307",266738,
        "Index",209923,
        "MailboxPurge",113300,
        "SessionSweeper",2460,
        "Junk-NotJunk-Handler",1201,
        "Timer-Zimbra",643,
        "ScalConnChecker",178,
        "LicenseCounter",22,
        "FileLogWriter.FsyncThread",12,
        "ScheduledTask",3,
        "SyncGalGroup",2]},